### PR TITLE
Updated the influx library to the version 5.9.3 from 5.0.7

### PR DESCRIPTION
**Pull request by Eezi**

### CODE REVIEW

File: packages/metrics/src/features/metrics/metricsGenerator.ts [hunk 1]
```diff
@@ -243,7 +251,14 @@ export async function fetchRegWithinTimeFrames(
     )
   }
 
-  const timeFramePoints = await query(queryString)
+  const timeFramePoints = await query(queryString, {
+    placeholders: {
+      timeStart,
+      timeEnd,
+      locationId,
+      registrationTargetInDays
```
Comment (Zangetsu101) [inline packages/metrics/src/features/metrics/metricsGenerator.ts:259]: I couldn't seem to find `registrationTargetInDays` in the query?

File: packages/metrics/src/features/metrics/metricsGenerator.ts [hunk 2]
```diff
@@ -399,12 +421,18 @@ export async function fetchKeyFigures(
...
+          OR locationLevel3 = $queryLocationId
+          OR locationLevel4 = $queryLocationId
+          OR locationLevel5 = $queryLocationId )
       AND ageInDays <= ${EXPECTED_BIRTH_REGISTRATION_IN_DAYS}
-    GROUP BY gender`
+    GROUP BY gender`,
+    {
+      placeholders: {
+        queryLocationId,
+        EXPECTED_BIRTH_REGISTRATION_IN_DAYS
```
Comment (Zangetsu101) [inline packages/metrics/src/features/metrics/metricsGenerator.ts:433]: I don't think we need `EXPECTED_BIRTH_REGISTRATION_IN_DAYS` inside the placeholders as it's still being directly interpolated in the string literal.

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/metrics/src/features/metrics/metricsGenerator.ts (hunk 1 above)
Comment (Eezi) [reply packages/metrics/src/features/metrics/metricsGenerator.ts:259]: Well noticed!

File: packages/metrics/src/features/declarationsStarted/service.ts [hunk 3]
```diff
@@ -157,16 +196,25 @@ export async function getAvgTimeSpentOnAppByPractitioners(
...
+                        OR locationLevel5 = $locationId ) 
+                    $eventClause
+                    GROUP BY practitionerId`,
+    {
+      placeholders: {
+        timeFrom,
+        timeTo,
+        locationId,
+        eventClause,
+        event
```
[the remark this answers is not in the document]
Comment (Eezi) [reply packages/metrics/src/features/declarationsStarted/service.ts:215]: You are totally right

File: packages/metrics/src/features/getTimeLogged/utils.ts [hunk 4]
```diff
@@ -53,18 +64,28 @@ export async function getTimeLoggedForPractitioner(
...
+            OR locationLevel5 = $locationId)
+              ORDER BY time DESC
+              ${countLimit}`,
+    {
+      placeholders: {
+        timeFrom,
+        timeTo,
+        practitionerId,
+        locationId
+      }
```
[the remark this answers is not in the document]
Comment (Eezi) [reply packages/metrics/src/features/getTimeLogged/utils.ts:86]: Yea true, I will add that.

**Comment by Eezi** [comment]
Thx for the review! I added all your recommend changes. I also committed and pushed the yarn.lock file changes, should remove this or just keep it?

**Comment by Zangetsu101** [comment]
> Thx for the review! I added all your recommend changes. I also committed and pushed the yarn.lock file changes, should remove this or just keep it?

It's fine. Let's keep it as it is.

**Review by Zangetsu101 — APPROVED** [review]
@Eezi Great work! Really appreciate the hard work you've done here.

**Comment by Zangetsu101** [comment]
@euanmillar We will need to QA the performance dashboard when we merge this in.

**Comment by euanmillar** [comment]
@Eezi thanks for your hard work on this!  We are going to put this into the v1.3.0 release as we will have to regression test a lot.

**Pull request merged**